lib/gui/app/styled-components.js

```diff
@@ -16,34 +16,70 @@
 
 'use strict'
 
-const styled = require('styled-components').default
-const { colors } = require('./theme')
+// eslint-disable-next-line no-unused-vars
+const React = require('react')
+const { default: styled } = require('styled-components')
 const {
-  Button, Txt, Flex
+  Button,
+  Txt,
+  Flex,
+  Provider
 } = require('rendition')
+const {
+  height,
+  space
+} = require('styled-system')
```

Comment:
I didn't check, so this is just to confirm, do you have `styled-system` as a dependency or you rely on rendition for it to be there?

Reply:
@sradevski Confirmed, `"styled-system": "^4.1.0"`